controlplane/src/core/bufservices/PlatformService.ts

```diff
@@ -699,6 +710,15 @@ export default function (opts: RouterOptions): Partial<ServiceImpl<typeof Platfo
               });
             }
           }
+
+          if (federatedGraph) {
+            orgWebhooks.send('graph.schema.updated', {
```

Comment:
how about this:

Add a map of eventName -> { event zod schema, event description }.
Add a version suffix to the eventName, e.g. "graph.schema.updated.v1"
Add a description for each event so that the generated docs are more meaningful.

Add a build time step that compiles the map into a README which defines all events and the JSON Schema for each event, so that we can link it in the docs.

In a later step, we could also provide an SDK in JS to handle the events with the eventName as the discriminator, but this is an extra.

The general idea is to not just create event handlers hidden in code but to also properly document all events from the beginning and give the idea of event handling a proper structure to define and document all events.

Reply:
In addition, the zod schema can be used to validate the data before we send it to the webhook, ensuring that we don't break the contract.

Sidenote, there's a library to generate a JSON Schema from zod.

Reply:
I have added version as a field. Docs will be updated manually for now